documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -481,11 +489,32 @@ spec:
     # ...
 ----
 
-When metrics are enabled, they are exposed on port 9404.
+.Using Strimzi Metrics Reporter
+The `metricsConfig` property contains configurations for the {StrimziMetricsReporter}.
+When configured to use Strimzi Metrics Reporter, Strimzi exposes the metrics provided by Apache Kafka directly.
```

Comment:
I do not think this means anything ... what exactly is _direct_ here?

Reply:
I think for user documentation, that is meaningless. If you want to explain the difference, you should focus on something that explains the value for them.

Reply:
new attempt: The Strimzi Metrics Reporter offers a lightweight solution for exposing Kafka metrics in Prometheus format, eliminating the need for intermediate layers like JMX or Java agents, and avoiding complex mapping rules that can introduce latency.

Reply:
What about leaving out the `eliminating the need for intermediate layers like JMX or Java agents` part? But yes, I think this looks reasonable.